chat-plugins/mafia.js

```diff
@@ -686,12 +685,12 @@ class Mafia extends Rooms.RoomGame {
 						} else {
```

Comment:
um, three more lines up, what is that for-loop trying to do? it looks like a bug

Reply:
Okay I thought more about what's going on. You should define `otherVoters = voters.filter(otherVoter => otherVoter !== voter);`.

That'll make the rest of this code extremely simple.

Reply:
Also OXFORD COMMA, WE'RE NOT SAVAGES

Reply:
i waSNT THE ONE WHO WROTE THE CHAT PLUGIN